src/amo/components/InstallWarning/index.js

```diff
@@ -61,13 +69,18 @@ export class InstallWarningBase extends React.Component<InternalProps> {
       userAgentInfo,
     });
 
+    const promotedCategory = _getPromotedCategory({
+      addon,
+      clientApp,
+    });
+
     return _couldShowWarning
       ? _couldShowWarning()
       : !correctedLocation &&
           isFirefox({ userAgentInfo }) &&
           clientApp === CLIENT_APP_FIREFOX &&
           addon.type === ADDON_TYPE_EXTENSION &&
-          !addon.isRecommended;
+          (!promotedCategory || promotedCategory === STRATEGIC);
```

Comment:
I wonder if this would be safer if it's a condition on the specific categories that override the warning, rather than assuming that all categories override the warning except STRATEGIC.

Reply:
Good idea!